config/application.rb

```diff
@@ -17,7 +17,7 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
-module WebsiteResume
+module NPCme
```

Comment:
So many little typos hanging around from the template I used!  One day I hope to find them all :)